symtabAPI/src/dwarfWalker.h

```diff
@@ -400,8 +411,8 @@ class DwarfWalker : public DwarfParseActions {
             Dwarf_Sword listLength);
 
 
-    // Header-only functions get multiple parsed.
-    std::set<FunctionBase *> parsedFuncs;
+    // Map of Function* to bool (indicates function parsed)
+    std::shared_ptr<ParsedFuncs> parsedFuncs;
```

Comment:
This has to be a `boost::shared_ptr`. Sigh. Maybe one day we'll adopt a C++ standard that's not a decade old. Also, I'm assuming this is shared not because of ownership but for copy semantics (i.e., we don't want to make DwarfWalker move-only by using a unique_ptr here)?